yt_dlp/extractor/scrolller.py

```diff
@@ -0,0 +1,101 @@
+from .common import InfoExtractor
+from ..utils import (determine_ext, KNOWN_EXTENSIONS)
+import json
+
+
+class ScrolllerIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?scrolller\.com/(?P<id>[\w-]+)'
+    _TESTS = [{
+        'url': 'https://scrolller.com/a-helping-hand-1k9pxikxkw',
+        'info_dict': {
+            'id': 'a-helping-hand-1k9pxikxkw',
+            'ext': "mp4",
+            'thumbnail': 'https://zepto.scrolller.com/a-helping-hand-3ty9q8x094-540x960.jpg',
+            'title': 'A helping hand',
+            "age_limit": 0,
+        }
+    }, {
+        'url': 'https://scrolller.com/tigers-chasing-a-drone-c5d1f2so6j',
+        'info_dict': {
+            "id": "tigers-chasing-a-drone-c5d1f2so6j",
+            'ext': "mp4",
+            'thumbnail': 'https://zepto.scrolller.com/tigers-chasing-a-drone-az9pkpguwe-540x303.jpg',
+            "title": "Tigers chasing a drone",
+            "age_limit": 0,
+        }
+    }, {
+        'url': 'https://scrolller.com/baby-rhino-smells-something-9chhugsv9p',
+        'info_dict': {
+            "id": "baby-rhino-smells-something-9chhugsv9p",
+            'ext': "mp4",
+            "thumbnail": 'https://atto.scrolller.com/hmm-whats-that-smell-bh54mf2c52-300x224.jpg',
+            "title": "Baby rhino smells something",
+            "age_limit": 0,
+        }
+    }, {
+        'url': 'https://scrolller.com/its-all-fun-and-games-cco8jjmoh7',
+        'info_dict': {
+            "id": "its-all-fun-and-games-cco8jjmoh7",
+            'ext': "mp4",
+            'thumbnail': 'https://atto.scrolller.com/its-all-fun-and-games-3amk9vg7m3-540x649.jpg',
+            "title": "It\'s all fun and games...",
+            "age_limit": 0,
+        }
+    }, {
+        'url': 'https://scrolller.com/may-the-force-be-with-you-octokuro-yeytg1fs7a',
+        'info_dict': {
+            "id": "may-the-force-be-with-you-octokuro-yeytg1fs7a",
+            'ext': "mp4",
+            'thumbnail': 'https://thumbs2.redgifs.com/DarkStarchyNautilus-poster.jpg',
+            "title": "May the force be with you (Octokuro)",
+            "age_limit": 18,
+        }
+    }
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        query = {
+            'query':
+            '''{
+                getSubredditPost(url:"/%s"){
+                    id
+                    title
+                    subredditTitle
+                    subredditUrl
+                    redditPath
+                    isNsfw
+                    mediaSources{
+                        url
+                        width
+                        height
+                    }
+                }
+            }''' % (video_id)
+        }
+
+        video = self._download_json(
```

Comment:
rename this to json_data or video_json or video_data or info_data (preferred) 

Naming it video is kinda confusing at first glance